automerge-wasm/src/interop.rs

```diff
@@ -457,3 +458,143 @@ pub(crate) fn list_to_js_at(doc: &am::AutoCommit, obj: &ObjId, heads: &[ChangeHa
     }
     array.into()
 }
+
+/*
+pub(crate) fn export_values<'a, V: Iterator<Item = Value<'a>>>(val: V) -> Array {
+  val.map(|v| export_value(&v)).collect()
+}
+*/
+
+pub(crate) fn export_value(val: &Value<'_>) -> JsValue {
+    match val {
+        Value::Object(o) if o == &am::ObjType::Map || o == &am::ObjType::Table => {
+            Object::new().into()
+        }
+        Value::Object(_) => Array::new().into(),
+        Value::Scalar(v) => ScalarValue(v.clone()).into(),
+    }
+}
+
+pub(crate) fn apply_patch(obj: JsValue, patch: &Patch) -> Result<JsValue, JsValue> {
+    apply_patch2(obj, patch, 0)
```

Comment:
Do we need `apply_patch2` or can we just consolidate this to `apply_patch`?

Reply:
I clean this up in the pojo_js pull request